cli/user/src/main/java/io/pravega/cli/user/config/InteractiveConfig.java

```diff
@@ -57,16 +57,26 @@ public class InteractiveConfig {
     private Level logLevel;
 
     public static InteractiveConfig getDefault() {
+        String controllerURI = "localhost:9090";
+        boolean tlsEnabled = false;
+        //Default tls based configurations for pravega cli
+        if (System.getenv("PRAVEGA_CONTROLLER_URI") != null) {
```

Comment:
is there any possibility of getting `System.getenv("PRAVEGA_CONTROLLER_URI")` as blank or any junk value?

Reply:
Yes, may be for some client this value might not be set, for such cases we have added it.

Reply:
In that case wouldn't it be good to add a check for empty string as well?

Reply:
If it will be there then it will always have the correct value present.